### PR TITLE
Fixed #32057 -- Doc'd HttpResponse.get()/items().

### PR DESCRIPTION
**Pull request by MAHANTH-wq**
I have added documentation for HttpResponse objects methods get(headers,alternate=None) and also items().

I'm new to the community so please help me out if i did any mistakes(This will be my first small contribution towards the community)
I don't know whether i have attached the issue to the pull requests ,if not please anyone guide me to do so.

### CODE REVIEW

File: docs/ref/request-response.txt [hunk 1]
```diff
@@ -820,6 +820,15 @@ Methods
...
     Returns the value for the given header name. Case-insensitive.
 
+.. method:: HttpResponse.get(header,alternate=none)
+    
+    Returns the value for the given header, or returns alternate if header
+    doesn't exist.
+
+.. method:: HttpResponse.items()
+
+    Returns all the tuples in the response's headers.
```
Comment (felixxm) [inline docs/ref/request-response.txt:830]: Maybe:
```suggestion
    Acts like :meth:`dict.items` for :attr:`HttpResponse.headers`.
```

**Review by felixxm — COMMENTED** [review]
@MAHANTH-wq Thanks :+1: I left suggestions.

File: docs/ref/request-response.txt (hunk 1 above)
Comment (MAHANTH-wq) [reply docs/ref/request-response.txt]: I have made the changes Sir..

File: docs/ref/request-response.txt [hunk 2]
```diff
@@ -820,6 +820,15 @@ Methods
 
     Returns the value for the given header name. Case-insensitive.
 
+.. method:: HttpResponse.get(header,alternate=none)
+    
+    Returns the value for the given header, or returns alternate if header
+    doesn't exist.
```
[the remark this answers is not in the document]
Comment (MAHANTH-wq) [reply docs/ref/request-response.txt]: Thanks sir, I have made the changes

File: docs/ref/request-response.txt [hunk 3]
```diff
@@ -820,6 +820,15 @@ Methods
 
     Returns the value for the given header name. Case-insensitive.
 
+.. method:: HttpResponse.get(header,alternate=none)
```
[the remark this answers is not in the document]
Comment (MAHANTH-wq) [reply docs/ref/request-response.txt]: Thanks for the support sir..

**Comment by MAHANTH-wq** [comment]
Sir, I own the ticket in code.djangoproject.com ,will merging this pull request will close that Ticket 32057 automatically or anything else should be done

**Comment by felixxm** [comment]
@MAHANTH-wq Thanks :+1: Welcome aboard :boat: 

> ... will merging this pull request will close that Ticket 32057 automatically or anything else should be done

Yes, the ticket will be closed automatically.

**Comment by MAHANTH-wq** [comment]
> @MAHANTH-wq Thanks 👍 Welcome aboard ⛵
> 
> > ... will merging this pull request will close that Ticket 32057 automatically or anything else should be done
> 
> Yes, the ticket will be closed automatically.

Okay sir,Thank you

**Review by carltongibson — APPROVED** [review]
LGTM. Thanks @MAHANTH-wq!

**Pull request merged**